x-pack/docs/en/rest-api/security/saml-invalidate-api.asciidoc

```diff
@@ -0,0 +1,87 @@
+[role="xpack"]
+[[security-api-saml-invalidate]]
+=== SAML invalidate API
+
+Submits a SAML LogoutRequest message to {es} for consumption. 
```

Comment:
As above, some text for _"you are only interested in this page if you are writing a web application that wants to use SAML authentication against ES"_.